Engine/Source/Actor/Material/ThinFilm/InterferenceTable.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <vector>
+
+namespace ph
+{
+
+/*
+	A table that records spectral reflectance and transmittance for each
+	incident angle. 
+
+	Reference:
+
+	Sussenbach, M., "Rendering Iridescent Objects in Real-time", 
+	Utrecht University Master Thesis (2013)
+
+	https://dspace.library.uu.nl/handle/1874/287110
+*/
+class InterferenceTable
+{
+public:
+	InterferenceTable();
+
+	// TODO
+
+private:
```

Comment:
You can store the calculated results here, just like how you populated the files.